tests/docker/README.md

```diff
@@ -0,0 +1,28 @@
+Docker setup for tests
+======================
+
+This docker setup allows you to run tests for this repository in a pre-configured environement.
+
+Start the containers and get a shell into the PHP container:
+```shell
+$ docker-compose up --build -d
+Recreating docker_db_1 ... done
+Recreating docker_web_1 ... done
+
+$ docker exec -it docker_web_1 bash
```

Comment:
Would change this to `docker-compose exec web bash`

Reply:
Yay, learned something!